ztp/source-crs/AmqOperatorStatus.yaml

```diff
@@ -0,0 +1,26 @@
+# This CR verifies the installation/upgrade of the Amq Operator
```

Comment:
These are all copies of each other, which seems like a tricky thing to manage; Can we either auto-generate them at runtime, or auto-generate them at build time from a common template, instead of just copying them multiple times?